src/app/opt/patch.rs

```diff
@@ -22,11 +14,10 @@ pub enum Patch {
 	},
 	Replace {
 		path: String,
-		value: Value,
+		value: String,
 	},
 	Change {
 		path: String,
-		#[serde(with = "DiffDef")]
-		diff: Diff,
+		value: String,
```

Comment:
This too...

Reply:
This should accept a string, have a look at this example from how the patch change operation works in SurrealQL:

```sql
update person:1 patch [{ op: "change", path: "/name", value: "@@ -1,4 +1,4 @@\n te\n-st\n+mp\n" }];
```